assets/adr.scss

```diff
@@ -67,14 +67,47 @@
   }
 
   /* Styling for the filter buttons */
-  .filter-section {
-    .filter-list {
-      list-style: none;
-      display: flex;
-      flex-wrap: wrap;
-      column-gap: 10px;
-      padding: 0;
-      .filter-button {
+  .adr-controls {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    .filter-section,
+    .sort-section {
+      .filter-list {
+        list-style: none;
+        display: flex;
+        flex-wrap: wrap;
+        column-gap: 10px;
+        padding: 0;
+        .filter-item {
+          margin: 0;
+        }
+      }
+      .sort-list,
+      .status-filter-list {
+        list-style: none;
+        border: 1px solid $white-smoke;
+        position: absolute;
+        background-color: $white;
+        border-radius: 4px;
+        margin-top: 8px;
+        box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
+        padding: 10px;
+        width: 40%;
+        z-index: 1;
+        .sort-item,
+        .status-filter-item {
+          cursor: pointer;
+          &.active {
+            font-weight: bold;
+          }
+        }
+      }
+      .status-filter-list {
```

Comment:
Possible to add stylelint here so that formatting for SCSS is taken care off?

Reply:
@swarad07 Thank you for the suggestion. I'll be using stylelint henceforth.